Licence_CeCILL_V2.1-en.txt

```diff
@@ -20,7 +20,7 @@ the two main principles guiding its drafting:
 The authors of the CeCILL (for Ce[a] C[nrs] I[nria] L[ogiciel] L[ibre])
 license are:
 
-Commissariat � l'�nergie atomique et aux �nergies alternatives - CEA, a
+Commissariat à l'énergie atomique et aux énergies alternatives - CEA, a
```

Comment:
Why do I get a change here? Encoding problem. Thanks to fix this please.

Reply:
The modification is the fix, if I'm not mistaken.

Reply:
But there is no relation with the PR and, on github, it is printed "Ã  l'Ã©nergie atomique et aux Ã©nergies" which is not readable. So @yannl35133 is it a mistake or is it on purpose?

Reply:
The file that I have is not Windows 1252 but ISO-8859:
```
15:44 ~/lambdapi (master) file Licence_CeCILL_V2.1-en.txt 
Licence_CeCILL_V2.1-en.txt: ISO-8859 text
```

Reply:
I checked in a hex editor : é is encoded here as E9 which is Latin1. The encoding is different in utf-8, it should be C3 A9